docs/_docs/reference/changed-features/pattern-matching.md

```diff
@@ -13,20 +13,27 @@ Scala 3 supports a superset of Scala 2 [extractors](https://www.scala-lang.org/f
 Extractors are objects that expose a method `unapply` or `unapplySeq`:
 
 ```scala
-def unapply[A](x: T)(implicit x: B): U
-def unapplySeq[A](x: T)(implicit x: B): U
+def unapply[A](x: T): U
+def unapplySeq[A](x: T): U
+```
+
```

Comment:
Type params are part of the general signature. It would be nice to say a word about `T` and `U`. Do they say "scrutinee"? "`U` determines how pattern matching is performed, as described below."

Reply:
There is also no requirement on type parameters (TODO: check), I think I will remove them from the first example

The type T can be arbitrary, I'll add some text about it, and explain that U is defined below

Reply:
I added some clarification at line 20